### PR TITLE
Testing to do list

### PR DESCRIPTION
**Pull request by AlexRS90**
## For this milestone we:
- Installed jest and babel libraries. 📚 
- Added test for adding new task functionality. 
- Added test for delete task functionality.

### CODE REVIEW

File: src/__tests__/task.test.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        index: 1,
+        description: 'task 1',
+        completed: false,
+      },
+      {
+        index: 2,
+        description: 'task 2',
+        completed: false,
+      },
+    ];
```
Comment (mikethreels) [inline src/__tests__/task.test.js:36]: - [ ] Good job on the tests, they are passing which is great! 💪 however in the tests you are creating the "todo" variable every time you run a test that is not according to the DRY rules, you can create the variable once in the `description()` method instead.

File: src/__tests__/task.test.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+ */
+
+import { deleteTodo } from '../task';
+import { getTodoList, saveTodoList } from '../functions';
+
+describe('add and delete tasks', () => {
+  test('adds a single to-do item to the list', () => {
+    const todo = [];
+    saveTodoList(todo); // save the empty array of to-do list items
+    const numberOfItemsBeforeSave = getTodoList().length;
```
Comment (mikethreels) [inline src/__tests__/task.test.js:12]: - [ ] good job on wrapping the tests inside the describe function 👍 According to the requirements however you should create mocks for functions that use `localStorage` for example.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/__tests__/task.test.js (hunk 1 above)
Comment (AlexRS90) [reply src/__tests__/task.test.js]: Thanks for your suggestion, we have fixed it.

File: src/__tests__/task.test.js (hunk 2 above)
Comment (AlexRS90) [reply src/__tests__/task.test.js:14]: Sure, we implemented mocks!

**Review by mikethreels — APPROVED** [review]
Hi team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 :
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**